tests/unit/tests/Donations/Models/TestDonation.php

```diff
@@ -121,4 +121,75 @@ public function testDonationShouldGetSequentialId()
 
         $this->assertEquals(1, $donation->getSequentialId());
     }
+
+    /**
+     * @unreleased
+     */
+    public function testDonationShouldReturnAmountInBaseCurrency()
+    {
+        $donation = $this->createDonation([
+            'amount' => new Money(5000, 'EUR'),
+            'exchangeRate' => '0.9',
+        ]);
+
+        self::assertMoneyEquals(
+            $donation->amount->inBaseCurrency($donation->exchangeRate),
+            $donation->amountInBaseCurrency()
+        );
+    }
+
+    /**
+     * @unreleased
+     */
+    public function testDonationShouldGetIntendedAmount()
+    {
+        // When a donation has a fee recovery amount
+        $donation = $this->createDonation([
+            'amount' => new Money(5000, 'USD'),
+            'feeAmountRecovered' => new Money(500, 'USD'),
+        ]);
+
+        self::assertMoneyEquals(new Money(4500, 'USD'), $donation->intendedAmount());
+
+        // When a donation does not have a fee recovery amount
+        $donation = $this->createDonation([
+            'amount' => new Money(5000, 'USD'),
+        ]);
+
+        self::assertMoneyEquals(new Money(5000, 'USD'), $donation->intendedAmount());
+    }
+
+    /**
+     * @unreleased
+     */
+    public function testDonationShouldGetIntendedAmountInBaseCurrency()
+    {
+        // When a donation has a fee recovery amount
+        $donation = $this->createDonation([
+            'amount' => new Money(5000, 'USD'),
+            'feeAmountRecovered' => new Money(500, 'USD'),
+            'exchangeRate' => '0.9',
+        ]);
+
+        self::assertMoneyEquals(
+            $donation->intendedAmount()->inBaseCurrency($donation->exchangeRate),
+            $donation->intendedAmountInBaseCurrency()
+        );
+    }
+
+    /**
+     * @unreleased
+     *
+     * @param array $attributes
+     *
+     * @return Donation
+     */
+    private function createDonation(array $attributes = [])
+    {
+        if (!isset($attributes['donorId'])) {
+            $attributes['donorId'] = Donor::factory()->create()->id;
+        }
+
+        return Donation::factory()->create($attributes);
+    }
```

Comment:
@JasonTheAdams you don't need to do this (anymore), I added relationship factory creations by default within the model factories recently.  So unless you specify a `donorId` the factory will create a donor for you.

Reply:
Lovely! Resolved in 8ada907